examples/rhino-sdk/federated_join/Federated Join Tutorial.ipynb

```diff
@@ -0,0 +1,585 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 77,
+   "id": "3b107de9",
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    },
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Logging In\n",
+      "········\n",
+      "Logged In\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
+    "import sys \n",
+    "from getpass import getpass, getuser\n",
+    "\n",
+    "import rhino_health # Only need this in final version\n",
+    "\n",
+    "print(\"Logging In\")\n",
+    "my_username = \"my_email@example.com\" # Replace this with the email you use to log into Rhino Health\n",
+    "my_password = getpass()\n",
+    "rhino_api_url= rhino_health.ApiEnvironment.PROD_API_URL\n",
+    "session = rhino_health.login(username=my_username, password=my_password, rhino_api_url=rhino_api_url, show_traceback=True)\n",
+    "\n",
+    "print(\"Logged In\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "59adf095",
+   "metadata": {},
+   "source": [
+    "# Intersection Mode\n",
```

Comment:
Tell me what Intersection Mode is and where I can learn more if applicable (links to ZenDesk/SDK docs)